MAUI/DataGrid/data-binding.md

```diff
@@ -0,0 +1,116 @@
+             x:Class="GettingStarted.MainPage">
+
+    <ContentPage.BindingContext>
+        <local:ViewModel/>
+    </ContentPage.BindingContext>
+
+    <syncfusion:SfDataGrid x:Name="sfDataGrid" ItemsSource="{Binding OrderInfoCollection}" />
+</ContentPage>
+{% endhighlight %}
+
+{% highlight c# %}
+OrderInfoViewModel orderInfoViewModel = new OrderInfoViewModel();
+this.sfDataGrid.ItemsSource = orderInfoViewModel.OrderInfoCollection;
+{% endhighlight %}
+{% endtabs %}
+
+If the data source implements [ICollectionChanged](https://docs.microsoft.com/en-us/dotnet/api/system.collections.icollection?view=net-6.0) interface, then the `SfDataGrid` will automatically refresh the view when an item is added, removed, or cleared. When you add or remove an item in [ObservableCollection](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.observablecollection-1?view=net-6.0), it automatically refreshes the view as the ObservableCollection. That implements the [INotifyCollectionChanged](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.observablecollection-1.system-componentmodel-inotifypropertychanged-propertychanged?view=net-6.0). But when you do the same in `List`, it will not refresh the view automatically.
+
+If the data model implements the `INotifyPropertyChanged` interface, then the SfDataGrid responds to the property change at runtime to update the view.
+
+## Binding with IEnumerable
+
+The `SfDataGrid` control supports to bind any collection that implements from [IEnumerable](https://docs.microsoft.com/en-us/dotnet/api/system.collections.ienumerable?view=net-6.0) interface. All the data operations such as sorting and filtering are supported when binding collection derived from IEnumerable.
+
+## Binding with DataTable
+
+The `SfDataGrid` control supports to bind the [DataTable](https://docs.microsoft.com/en-us/dotnet/api/system.data.datatable?view=net-6.0). `SfDataGrid` control automatically refresh the UI when binding DataTable as ItemsSource when rows are added, removed or cleared.
+
+{% tabs %}
+{% highlight c# %}
+DataTable table = this.GetDataTable();
+this.sfDataGrid1.ItemsSource = table;
+{% endhighlight %}
+{% endtabs %}
+
+### Limitations
+
+* Custom sorting is not supported.
+* [SfDataGrid.View.Filter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_Filter) is not supported.
+* [SfDataGrid.View.LiveDataUpdateMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_LiveDataUpdateMode) is not supported.
+
+## View
+
+The `SfDataGrid` has the View property of type [ICollectionViewAdv](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html) interface that implements `ICollectionView` interface. View is responsible for maintaining and manipulating data and other advanced operations, like Sorting, Filtering, and etc.,

```

Comment:
The `SfDataGrid` has the View property of type [ICollectionViewAdv](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html) interface that implements the `ICollectionView` interface. The view is responsible for maintaining and manipulating data and other advanced operations, like Sorting, Filtering, etc.,

Reply:
Modified.